src/utils.js

```diff
@@ -76,7 +76,8 @@ exports.GetModule = (message, args, send = true) => {
 			result = GetMatching(args._.join(' '))
 			if (result && result.length > 0) return HandleRegex(result)
 			else {
-				if (send) message.channel.send(`🚫 Couldn't find a module by the ID of \`${args._[0]}\` (case-sensitive), name of \`${args._.join(' ')}\` (not case-sensitive) or periodic symbol of \`${args._[0]}\` (not case-sensitive)`)
+				const cleanseDiscordText = (text) => text.replace(/`/g, "");

```

Comment:
This shouldn't be defined every time the `GetModule` function is called

Reply:
Understandable